### DISCUSSION TITLE
On GitHub Desktop, is there a way to see the filepath and folder GitHub is using for a repository or branch on the desktop PC?

### DISCUSSION THREAD

**Discussion by git-along-little-doggie** [community/community#18353] [Other Feature Feedback, Questions, & Ideas] [answered]
On GitHub Desktop (or even GitHub in the browser), is there a way to see the filepath and folder on my desktop PC that GitHub is using for a repository or a branch?

If not, please could you add this feature?

(I am new to GitHub so please assume no prior knowledge. )

Also, for new users, when executing an action, some feedback in the interface in plain English would be helpful. Eg. "File A has been copied from your desktop_folder_name to GitHub cloud.", "File B has been copied from GitHub cloud to your desktop_folder_name." 

*What* happened is important but *where* it happened is also important. The latter seems to be missing.

**Comment by doguedogue** [comment, marked as answer]
Yes, you can
1. you select the specific repository you want to visualize selecting by a list
2. you can see in the path of your PC

![image](https://user-images.githubusercontent.com/23409026/173212837-e60a79c3-2246-4fc1-b109-02472229ccbf.png)

![image](https://user-images.githubusercontent.com/23409026/173212826-d4b96197-5316-45b1-9299-ebf29f4f5026.png)

**Reply by git-along-little-doggie** [reply]
Thank you.

**Reply by doguedogue** [reply]
I hope this help you, close your question, marking as answered please!
